app/controllers/avo/search_controller.rb

```diff
@@ -1,5 +1,7 @@
 require_dependency "avo/application_controller"
 
+include ActionView::Helpers::TextHelper
```

Comment:
We should add this include inside `Avo::SearchController`.

Context: https://stackoverflow.com/a/5161484/9067704

Reply:
I will update it. Thank you.